<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DDSMonitor</class>
 <widget class="QDialog" name="DDSMonitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1179</width>
    <height>846</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DDS Monitor</string>
  </property>
  <widget class="QTextBrowser" name="Monitor_TextBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>1141</width>
     <height>731</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;---DDS Monitor---&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ChooseDDS_ComboBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="ChooseDDS_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose DDS</string>
   </property>
  </widget>
  <widget class="QLabel" name="RefreshTime_label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh Time</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="RefreshTime_SpinBox">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Start_Button">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Stop_Button">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Current_Button">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Current</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
